### PR TITLE
Added a cmd to drop course users.

### PR DESCRIPTION
**Pull request by OliverLok**

### CODE REVIEW

File: cmd/course-user-drop/main.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	TargetEmail string `help:"Email of the user to drop." arg:""`
+	CourseID    string `help:"ID of the course." arg:""`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("Drop a user from a course."),
```
Comment (Lucas-Ellenberger) [inline cmd/course-user-drop/main.go:23]: /s/from a course/from the course

(Match the [python interface](https://github.com/edulinq/autograder-py/blob/9642fa80dd07925f94bed620fe12497524711f32/autograder/api/courses/users/drop.py#L15)).

**Review by Lucas-Ellenberger — CHANGES_REQUESTED** [review]
Looking great!

Nit on the description.

**Review by Lucas-Ellenberger — APPROVED** [review]
lgtm!

**Review by eriq-augustine — APPROVED** [review]
lgtm

**Pull request merged**